docs/rest_api.md

```diff
@@ -416,6 +416,12 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
```

Comment:
We should document what the valid body values (pxe, disk, none) mean. It's ok to say stuff like "For IPMI devices, ..."